src/norm/pragmas.nim

```diff
@@ -49,5 +49,5 @@ template onDelete*(val: string) {.pragma.}
 template onUpdate*(val: string) {.pragma.}
   ## Add an ``ON UPDATE <POLICY>`` constraint.
 
-template table*(val: string) {.pragma.}
+template dbTable*(val: string) {.pragma.}
```

Comment:
Please keep the old pragma bu mark it as deprecated. It's better to give Norm users some time to update their code.